lib/src/verisync_button.dart

```diff
@@ -118,24 +116,8 @@ class VerisyncButton extends StatelessWidget {
         clientId: clientId,
         email: email,
         metadata: metadata,
-        onSuccess: onSuccess ??
-            (BuildContext dialogContext) {
-              ScaffoldMessenger.of(dialogContext).showSnackBar(
-                const SnackBar(
-                  content: Text("Verification successful"),
-                  duration: Duration(seconds: 3),
-                ),
-              );
-            },
-        onError: onError ??
-            (BuildContext dialogContext) {
-              ScaffoldMessenger.of(dialogContext).showSnackBar(
-                const SnackBar(
-                  content: Text("Verification failed"),
-                  duration: Duration(seconds: 3),
-                ),
```

Comment:
We want to provide a default behaviour in case the onSuccess or onError callbacks are not provided. The idea is to have a default behaviour for success and failure.